Page/product/product.html

```diff
@@ -0,0 +1,268 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/style.css">
+    <link rel="stylesheet" href="product.css">
+    <script src="js/fontawesomepro.js"></script>
+    <link rel="stylesheet" href="https://site-assets.fontawesome.com/releases/v6.0.0/css/all.css">
+    <link rel="stylesheet" href="https://mango-ui.netlify.app/components/card/card.css" />
+    <link rel="stylesheet" href="https://mango-ui.netlify.app/components/button/button.css" />
```

Comment:
Inherit one single link from Component library instead of repeatedly linking every component.

Reply:
I wanted 2 particularly because it was hindering my CSS somehow

Reply:
@tushartiwari7 might be better to keep it as is, because in terms of performance, importing them directly here would be better